lib/http-context.js

```diff
@@ -192,27 +191,45 @@ function coerce(str) {
 // coerce every string in the given object / array
 function coerceAll(obj) {
   var type = Array.isArray(obj) ? 'array' : typeof obj;
+  var i;
+  var n;
 
   switch (type) {
     case 'string':
       return coerce(obj);
     case 'object':
       if (obj) {
-        Object.keys(obj).forEach(function(key) {
+        var props = Object.keys(obj);
+        for (i = 0, n = props.length; i < n; i++) {
```

Comment:
I don't know what the house style is but just you can just use `var i` here and below.  The declaration gets hoisted to the top and it doesn't matter from a performance perspective.